rust/src/operations/create.rs

```diff
@@ -333,8 +333,12 @@ mod tests {
     #[tokio::test]
     async fn test_create_local_relative_path() {
         let table_schema = get_delta_schema();
-        let name = Alphanumeric.sample_string(&mut rand::thread_rng(), 16);
-        let table = DeltaOps::try_from_uri(format!("./{}", name))
+        let tmp_dir = TempDir::new_in(".", "tmp_").unwrap();
```

Comment:
These tests were creating directories without cleaning them up.

Reply:
Thanks for cleaning that up!